apps/onboarding/src/gateway/captcha/captcha.service.spec.ts

```diff
@@ -13,9 +14,20 @@ describe('CaptchaService', () => {
     }).compile();
 
     service = module.get<CaptchaService>(CaptchaService);
+    httpService = module.get<HttpService>(HttpService);
   });
 
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  it('should verify the token', async() => {
```

Comment:
Nice! Can you add tests to the CaptchaRule class as well? It'll be more interesting when we have complex checks and logging to test for.